receiver/filereceiver/README.md

```diff
@@ -0,0 +1,28 @@
+# File Receiver
+
+| Status                   |               |
+|--------------------------|---------------|
+| Stability                | [development] |
+| Supported pipeline types | metrics       |
```

Comment:
Considering that the exporter emits `logs`, `metrics`, and `traces`, should this also support these pipelines?

Reply:
Hi @MovieStoreGuy, thanks for your feedback. I was thinking that although eventually we'll want the file receiver to be able to handle all three, for this initial version, it would just support metrics, and the other pipelines would be added later. Do you think we should indicate the additional pipelines as supported now?

Reply:
It seems worthwhile to include support for all types from the start, since there shouldn't be much additional work.